proposal/comp110_architecture_proposal.tex

```diff
@@ -17,58 +17,58 @@ \section*{Topic}
+\item[Web link:] \url{http://dl.acm.org/citation.cfm?id=1536548}
+\item[Full text link:] \url{https://users.soe.ucsc.edu/~ejw/papers/smith-platformer-generation-fdg2009.pdf}
+\item[Comments:] This is a conference article that discusses the possibilities of creating playable levels with the incorporation of rhythm generation for geometry placed platforms.
 \end{description}
 
 \section*{Paper 2}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] An approach to level design using procedural content generation and difficulty curves
+\item[Citation:] \cite{diaz:2013}
+\item[Abstract:] "Level design is an art which consists of creating the combination of challenge, competition, and interaction that players call fun and involves a careful and deliberate development of the game space. When working with procedural content generation, it is necessary to review how the game designer sets the change in difficulty throughout the different levels. In this paper we present a procedural level generator that can be used for different games and is based on a genetic algorithm. We define a fitness function that does not depend on the game or content type. This function calculates the difference between the difficulty curve defined by the designer and the difficulty curve calculated for the candidate content, so the best content is the one whose difficulty curve best fits the desired curve. To design our generator, we rely on the concept of flow, theories of fun and game design."
+\item[Web link:] \url{http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=6633640&queryText=procedural%20level%20generation%20for%202d%20platform&newsearch=true}
+\item[Full text link:] \url{http://www.ijcsit.com/docs/Volume%206/vol6issue02/ijcsit2015060227.pdf}
+\item[Comments:]  This article attempts to demonstrate how level creation should be generatored through functions that compare the intended difficultly by the desinger and the difficulty for the player content.
 \end{description}
 
 \section*{Paper 3}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] PCG-based game design: creating Endless Web
```

Comment:
This paper seems to build on Smith's earlier work (paper 1 above, and the corresponding 2011 journal article) by integrating the system in a game. As such, this paper will be useful supporting material when discussing that one.